### PR TITLE
Update ar.json

### PR DESCRIPTION
**Pull request by ghost**
Pushing new arabic translated strings for Mastodon web client.

### CODE REVIEW

File: app/javascript/mastodon/locales/ar.json [hunk 1]
```diff
@@ -57,26 +57,26 @@
...
+  "embed.instructions": "يمكنكم إدماج هذه الحالة على موقعكم الإلكتروني عن طريق نسخ الشفرة أدناه.",
+  "embed.preview": "هكذا ما سوف يبدو عليه :",
   "emoji_button.activity": "الأنشطة",
-  "emoji_button.custom": "Custom",
+  "emoji_button.custom": "مخصص",
   "emoji_button.flags": "الأعلام",
   "emoji_button.food": "الطعام والشراب",
   "emoji_button.label": "أدرج إيموجي",
   "emoji_button.nature": "الطبيعة",
   "emoji_button.not_found": "No emojos!! (╯°□°）╯︵ ┻━┻",
```
Comment (ykzts) [inline app/javascript/mastodon/locales/ar.json]: Is it unnecessary to translate this message?

Comment (BoFFire) [reply app/javascript/mastodon/locales/ar.json]: Let me check.

Comment (BoFFire) [reply app/javascript/mastodon/locales/ar.json]: It is in the emoji frame. What do you think ? :)

**Review by Gargron — APPROVED** [review]
Looks fine to me, though I can see some strings are still untranslated

File: app/javascript/mastodon/locales/ar.json (hunk 1 above)
Comment (BoFFire) [reply app/javascript/mastodon/locales/ar.json]: @ykzts if there is any problem, I'll revert the translation, no problem and keep the english one :)
@Gargron Thank you :) Yes I know, lack of time and vocabulary sometimes. So I prefer to take some time before pushing things. Hope that other localizers JoinMastodon :)
Salutations amicales.

**Review by ykzts — APPROVED** [review]
Yeah!

**Pull request merged**